opendevin/runtime/docker/ssh_box.py

```diff
@@ -738,21 +769,21 @@ def close(self):
     except Exception as e:
         logger.exception('Failed to start Docker container: %s', e)
         sys.exit(1)
-
+    logger.setLevel('DEBUG')
```

Comment:
Forget to delete or intentional?

Reply:
intentional; as it is for devs

Reply:
I am strongly against changing logger level in the code

Reply:
It is in the main block only.

Reply:
> It is in the main block only.

Ah I see, that's much less concerning then... I still prefer we don't do it in the code, coz it might surprise developers

Reply:
Maybe logging about the change would help?